unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -495,6 +526,7 @@ fn constraint(mut node: Pairs<Rule>) -> RuleFragment {
         Rule::list_constraint => list_constraint(inverted, child.into_inner()),
         Rule::hostname_constraint => hostname_constraint(inverted, child.into_inner()),
         Rule::external_value => external_value(inverted, child.into_inner()),
+        Rule::ip_constraint => ip_matching_constraint(child.into_inner()),
```

Comment:
We don't use invert for this strategy and I want to keep this simple until we can throw it away